libwget/html_url.c

```diff
@@ -66,8 +66,8 @@ static const char maybe[256] = {
 	['u'] = 1,
 };
 static const char attrs[][12] = {
-	"action", "archive",
-	"background",
+	"action", "archive",						// Add new entry in sorted position
+	"background",								// as binary search is performed on attrs[][]
 	"code", "codebase", "cite", "classid",
```

Comment:
Maybe you can put your comments directly before the declaration (static const...), they are a good addition.